src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Create.cs

```diff
@@ -25,15 +22,7 @@ internal interface IDeclaredSymbolInfoFactoryService : ILanguageService
     {
         // `rootNamespace` is required for VB projects that has non-global namespace as root namespace,
         // otherwise we would not be able to get correct data from syntax.
-        bool TryGetDeclaredSymbolInfo(StringTable stringTable, SyntaxNode node, string rootNamespace, out DeclaredSymbolInfo declaredSymbolInfo);
-
-        // Get the name of the receiver type of specified extension method declaration node.
-        // The returned value would be "" or "[]" for complex types.
-        string GetReceiverTypeName(SyntaxNode node);
-
-        bool TryGetAliasesFromUsingDirective(SyntaxNode node, out ImmutableArray<(string aliasName, string name)> aliases);
-
-        string GetRootNamespace(CompilationOptions compilationOptions);
+        void AddDeclaredSymbolInfos(Document document, SyntaxNode root, ArrayBuilder<DeclaredSymbolInfo> declaredSymbolInfos, Dictionary<string, ArrayBuilder<int>> extensionMethodInfo, CancellationToken cancellationToken);
```

Comment:
all the above methods still exist, but are only used int he impl of AddDeclaredSymbolInfos, so they dont' need be exposed externally.